public/doc/openapi-3-v1.0.0.json

```diff
@@ -456,6 +456,15 @@
                 "description": "Return all approval workflows in ascending sequence order",
                 "operationId": "listWorkflows",
                 "parameters": [
+                    {
```

Comment:
Shouldn't them be grouped under `remote_object_query_params` as stated in the PR description? Let's use the name `resource_object` instead to align with what is in the spec already.

Best if could reuse `ResourceObject` schema.